velox/exec/tests/TableWriteTest.cpp

```diff
@@ -2449,7 +2463,15 @@ TEST_P(AllTableWriterTest, tableWriteOutputCheck) {
       if (commitStrategy_ == CommitStrategy::kNoCommit) {
         ASSERT_EQ(writeFileName, targetFileName);
       } else {
-        ASSERT_TRUE(writeFileName.find(targetFileName) != std::string::npos);
+        std::string suffix = ".parquet";
+        if (folly::StringPiece(targetFileName).endsWith(suffix)) {
+          // Remove the .parquet suffix.
+          auto trimmedFilename =
+              targetFileName.substr(0, targetFileName.size() - suffix.size());
+          ASSERT_TRUE(writeFileName.find(trimmedFilename) != std::string::npos);
```

Comment:
Why do we trim the targetFileName here? writeFileName should also have the ".parquet" suffix?

Reply:
The `targetFileName `format is `XX_writeFileName_uuid.parquet`, while the `writeFileName `is simply `writeFileName.parquet`. Therefore, we need to remove the .parquet extension from the `writeFileName`.

Reply:
I see. This is to make `std::string.find()` happy. Thanks!